Net/src/SocketReactor.cpp

```diff
@@ -179,26 +179,28 @@ bool SocketReactor::hasEventHandler(const Socket& socket, const Poco::AbstractOb
 
 SocketReactor::NotifierPtr SocketReactor::getNotifier(const Socket& socket, bool makeNew)
 {
+	if (socket.impl() == nullptr) return 0;
```

Comment:
@micheleselea let's cache the `SocketImpl*` here and also the `poco_socket_t` below to avoid repeated function calls. I know they are inlines but inlining is not guaranteed

Reply:
Ok correct point, how can I make change, let me know if it's ok because I don't remember if I use correct procedure